frontend/src/components/hackathon-page.tsx

```
@@ -1,402 +1,507 @@
-                                                <ul className="space-y-2 text-gray-300">
-                                                    {section.content.map((bullet: string, bIndex: number) => (
-                                                        <li key={bIndex} className="flex items-start">
-                                                            <span className="text-hack-primary mr-2">•</span>
-                                                            <span className="flex-1">{bullet}</span>
-                                                        </li>
-                                                    ))}
-                                                </ul>
-                                            )}
-                                        </div>
-                                    ))}
+            {Schedule && Schedule.schedule && (
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)